tests/integration/_cases/logs/logs-list-with-zero-max-rows.trycmd

```diff
@@ -0,0 +1,3 @@
+$ sentry-cli logs list --org wat-org --project 12345 --max-rows 0
```

Comment:
imo, we should validate that users pass a value above 0 for `--max-rows` because it does not make sense to pass zero. In any case, I don't think this test is very helpful, as it does not really make any sense to do this